client/src/partials/templates/patientListV2.tmpl.html

```diff
@@ -0,0 +1,5 @@
+<a>
```

Comment:
Why not update the original patient list template file?  Why is a separate file necessary?